pkgs/applications/misc/1password-gui/default.nix

```diff
@@ -64,14 +95,10 @@ stdenv.mkDerivation rec {
     description = "Multi-platform password manager";
     longDescription = ''
```

Comment:
This seems to be duplicating the `description` now that the "disclaimer" is gone (woo!) -- I think it's fair to remove it :)